docs/reference/module-api/module-overview.md

```diff
@@ -483,18 +483,354 @@ module.exports = {
+
+The `helpers` section returns an object of functions that add template utility methods. The individual helper methods may take any arguments that you plan to pass them in templates.
+
+Helpers are called in templates from their module on the `apos` object. See the [`alias`](/reference/module-api/module-options.md#alias) option to make this less verbose.
+
+```javascript
+// modules/product/index.js
+module.exports = {
+  // ...
+  options: {
+    alias: 'product'
+  },
+  helpers(self) {
+    return {
+      formatPrice(product) {
+        const price = product.price;
+        return `$${floatPrice.toFixed(2)}`;
+      }
+    };
+  }
+};
+```
+
+Using in a template:
+
+```django
+{# modules/product-page/views/show.html #}
+{{ apos.product.discountPrice(data.piece) }}
+```
+
 #### `extendHelpers(self)`
-### `apiRoutes(self)`
-#### `extendApiRoutes(self)`
+
+Add to the functionality of a template helper inherited from the base class. This must return an object of functions, similar to [`helpers`](#helpers-self).
+
+Extended helpers should take a `_super` argument, followed by the normal arguments of the helper being extended. If the original helper took only a `price` argument, the extending function should take the arguments `_super, price`.
+
+To maintain the same application, they should return the same type of response as the original helper. If the original returned a string, the extension should return a string.
+
+```javascript
+// modules/featured-product/index.js
+module.exports = {
+  extend: 'product',
+  // ...
+  extendHelpers(self) {
+    return {
+      formatPrice(_super, product) {
+        const price = _super(product);
+        // 👇 Adds some extra flash to the featured product prices.
+        return `${price} 🤑`;
+      }
+    };
+  }
+};
+```
+
 ### `restApiRoutes(self)`
+
+Apostrophe includes a full REST API for [pieces](/reference/api/pieces.md) and [pages](/reference/api/pages.md). Those route handlers can be totally overridden by adding identically named functions to object returned by the `restApiRoutes` function. If you simply wish to add to the existing behavior of the REST API routes, see [`extendRestApiRoutes`](#extendrestapiroutes-self).
+
+REST API functions take the route request as an argument (`req`, below);
+Valid names for functions returned by `restApiRoutes` include:
+- `getAll`
+- `getOne`
+- `post`
+- `patch`
+- `put`
+- `delete`
+
+```javascript
+// modules/product/index.js
+module.exports = {
+  // ...
+  restApiRoutes(self) {
+    return {
+      // GET /api/v1/product
+      async getAll(req) {
+        const results = [];
+        // ... populate `results` with product data.
+
+        return {
+          results
+        };
+      }
+    };
+  }
+};
+```
+
 #### `extendRestApiRoutes(self)`
+
+Extend the behavior of existing REST API routes in the `extendRestApiRoutes` section. This function must return an object of functions. See [`restApiRoutes`](#restapiroutes-self) for the valid function names.
```

Comment:
"in the `extendRestApiRoutes` section" -> remove

Reply:
These should say "function" not "section."

Reply:
Or nothing.